### PR TITLE
Re-order nav links, update download box to include shapefiles, squishy pages

### LINKED ISSUE #1324: Make link to Github more prominent / easy to find - add to footer

### LINKED ISSUE #1425: Move About in the navigation & make Explore the Tool the default page

### LINKED ISSUE #1436: Add Public Engagement Calendar and RFI to the footer

### LINKED ISSUE #1532: Updated about page (squish)
Need to update the about page to include a link to the FAQ and add the FAQ to the footer.

**To Reproduce**
1. Go to 'About'
2. Scroll down to 'below About Justice40.'

**Expected behavior**
Add a link to the FAQ under the main about content and added to the footer as well.

**Screenshots**
About_squish

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### LINKED ISSUE #1533: Modified download box on methodology & data page (squish)
# Purpose
Modify download box with

- new copy
- new button

Screen Shot 2022-04-05 at 1 00 00 PM

### PR DESCRIPTION
**Pull request by vim-usds**
# Purpose
Re-order the navigation links. This PR creates all the placeholder logic to add dropdown navigation links in the future. The TSD, FAQs and download page are also ready to go when we are ready in the future

- closes #1425 
- closes #1533 
- closes #1532
- closes #1436 
- closes #1324 

### QA spec:

Add shapefile download to download box
- [ ] does download button work?
- [ ] how is the styling of the box?
- [ ] how do colors look? Blue public engagement button near the darker blue download box?
- [ ] placement of the new Tag?

Add FAQ link to About page
- [ ] the text has a minimum 92 character requirement. that's why the copy has been updated. Is it ok, or do you have something else that could also meet the 92 character limit?

## QA testing round 1

<details>

### QA spec:

Re-order the nav links:
- [x] does the map/explore page load by default?
- [x] do all the navigation links work?
- [x] do all (most) links work as expected on each page?

Engagement calendar button on each page
- [x] is there a public engagement button on each page?
- [x] does the button work as expected?

Add FAQ link to About page
- [x] does the FAQ link work?

Modify footer
- [x] is the title of the second column in the footer correct?
- [x] are all footer links in the second column correct?
- [x] are all the footer links in the second column in the correct order?
- [x] is the title of the third column in the footer correct?

Questions for QA:
1. ~We have 3 different buttons on the site. A blue with white text, white with dark text, and yellow with blue text. I have a feeling that having at most two buttons unless there's some reason to have 3 different types. Wondering if we can discuss the options and see if all 3 button types are needed or if they cause extra cognitive load?~
</details>

### CODE REVIEW

**Review by esfoobar-usds — APPROVED** [review]
Just a couple of notes 🍏

**Comment by KameronKerger** [comment]
Screen Shot 2022-04-06 at 1 42 49 PM

Find answers on the Climate and Economic Justice Screening Tool's Frequently Asked Questions.

**Comment by KameronKerger** [comment]
updated pic based on @katherinedm-usds's feedback attached
Screen Shot 2022-04-06 at 2 17 27 PM

**Comment by KameronKerger** [comment]
Screen Shot 2022-04-06 at 2 30 46 PM

File: client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap [hunk 1]
```diff
@@ -79,7 +79,30 @@ exports[`J40Footer renders correctly 1`] = `
                   class="usa-footer__secondary-link"
                 >
                   <a
-                    class="usa-link usa-link--external footer-link-first-child"
+                    class="footer-link-first-child"
+                    href="/en/public-engagement"
+                  >
+                    Engagement calender
```
[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap:85]: @esfoobar-usds - yes it is. So the i18n for this link occurs in the J40Footer component on [line 40](https://github.com/usds/justice40-tool/blob/87372126abf30478a2578c1d87af2fb409b7e448/client/src/components/J40Footer/J40Footer.tsx#L40).

What you've highlighted is the snapshot. The snapshot is the rendered DOM at build time. Currently our snapshots only test the English render. Adding snapshot testing to include Spanish would be a good addition.

File: client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap [hunk 2]
```diff
@@ -79,7 +79,30 @@ exports[`J40Footer renders correctly 1`] = `
...
+                  class="usa-footer__secondary-link"
+                >
+                  <a
+                    class="usa-link usa-link--external"
+                    data-cy="request-for-infomation"
+                    href="https://www.federalregister.gov/d/2022-03920"
+                    rel="noreferrer"
+                    target="_blank"
+                  >
+                    Request for Infomation
```
[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap:98]: @esfoobar-usds - yes it is. So the i18n for this link occurs in the J40Footer component on [line 49](https://github.com/usds/justice40-tool/blob/87372126abf30478a2578c1d87af2fb409b7e448/client/src/components/J40Footer/J40Footer.tsx#L49).

What you've highlighted is the snapshot. The snapshot is the rendered DOM at build time. Currently our snapshots only test the English render. Adding snapshot testing to include Spanish would be a good addition.

File: client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap (hunk 1 above)
[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap:85]: Validating that Spanish snapshots are valid would require Spanish fluency. So maybe that's something we can chat about on ways to integrate that. I'm not certain if Spanish snapshots are normally part of the i18n pipeline. I have to research this a bit more. 
Great question tho!

[the remark this answers is not in the document]
Comment (esfoobar-usds) [reply client/src/components/J40Footer/__snapshots__/J40Footer.spec.tsx.snap:85]: And awesome answer! Thanks!

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Pull request merged**